CHANGELOG.markdown

```diff
@@ -1,3 +1,9 @@
+5.5.3
+-----
+* Enabled `PolyKinds` extension. The following datatypes now have polymorphic
+  kinds: `Cayley`, `Procompose`, `Rift`, `ProfunctorFunctor`, `Ran`, `Codensity`,
+  `Prep`, `Coprep`, `Star`, `Costar`, `WrappedArrow`, `Forget`.
```

Comment:
I think you made certain things poly-kinded that aren't listed here (`(:->)`, for instance).

Reply:
I checked the rest and I think `:~>` is the only one I missed, all other type synonyms are monokinded.